### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by uniontech-lilinjie**
Signed-off-by: lilinjie lilinjie@uniontech.com
fix typos

### CODE REVIEW

File: man/po/da.po [hunk 1]
```diff
@@ -5471,7 +5471,7 @@ msgstr ""
 
 #: login.defs.5.xml:34(para) login.1.xml:34(para)
 msgid "Enable logging of successful logins."
-msgstr "Aktiver logning af succesfulde logind."
+msgstr "Aktiver logning af successfulde logind."
```
Comment (ikerexxe) [inline man/po/da.po:5474]: Was this change intentional? I'm not a Danish speaker so I'm not sure, but it seems like you replaced all `succesful` instances by `successful`

Comment (hallyn) [reply man/po/da.po:5474]: Looking at translate.google.com, it looks like succesfulde is 'success' while successfulde is successful.  So I'll take it these are intentional, and merge.

Thanks, @uniontech-lilinjie , and thanks for looking, @ikerexxe
